### PR TITLE
Improve docstring in IEC 61853 module temperature model

### LINKED ISSUE #750: Implement IEC 61853 module temperature model
The Faiman module temperature model is used in IEC 61853.  It conceptually not much different from others, but uses slightly different parameters.

### PR DESCRIPTION
**Pull request by adriesse**
- [x] Closes #750
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [ ] Tests added
 - [ ] Updates entries to [`docs/sphinx/source/api.rst`](https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/api.rst) for API changes.
 - [ ] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/master/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by adriesse** [comment]
@cwhanse Could you check this?

File: pvlib/temperature.py [hunk 1]
```diff
@@ -307,6 +307,10 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
 
     Notes
     -----
+    Faiman [1] uses the parameter names U'0 and U'1 in equation 5, which
```
Comment (cwhanse) [inline pvlib/temperature.py]:
```suggestion
    Faiman [1] uses the parameter names U'_0 and U'_1 in equation 5, which
```

**Comment by wholmgren** [comment]
Just to confirm... Are you happy with the way the docstring is rendered? See the `continuous-documentation/read-the-docs` link in the checks. It's not what I'd choose to do but I'm ok merging.

**Comment by adriesse** [comment]
Sorry I don't see that link and I don't know how the doc string looks.  :(

**Comment by wholmgren** [comment]
Do you see it if you click "show all checks"?

Screen Shot 2019-12-19 at 10 52 21 AM

Screen Shot 2019-12-19 at 10 53 55 AM

**Comment by adriesse** [comment]
Thanks, yes, not nice.  I would remove the three underscores tomorrow unless Cliff objects.

Also, the units for u1  and u_v in pvsyst_cell look suspicious...

**Comment by kanderso-nrel** [comment]
@adriesse `...in equation 5 of [1]_...` is rendering the underscore because the citation it references isn't formatted in a way that numpydoc recognizes.  Here's an example of compatible reference formatting (see the leading two dots) [example](https://github.com/pvlib/pvlib-python/blob/master/pvlib/temperature.py#L104).

You might also want to use sphinx's `:math:` notation to interpret U’_0 and U’_1 as Latex code instead of plaintext: [example](https://github.com/pvlib/pvlib-python/blob/master/pvlib/temperature.py#L211)

**Comment by cwhanse** [comment]
> Thanks, yes, not nice.  I would remove the three underscores tomorrow unless Cliff objects.

I don't object to dropping the first two, which would format as subscripts if we added the math keyword but I don't think that's needed.

The third underscore needs to stay and the references need an adjusted format #833

**Comment by adriesse** [comment]
I understand, and I appreciate the hints all around, but this simple clarification is again turning into way more than I bargained for.

**Pull request closed without merging**